docs/guide/image_editor.md

```diff
@@ -0,0 +1,82 @@
+# Configuring the Image Editor
```

Comment:
Please add an Experience/Commerce badge, as https://github.com/ezsystems/developer-documentation/pull/1230 has been merged and image editor is in those two editions only.